python/python/glide/async_commands/core.py

```diff
@@ -2888,6 +2888,38 @@ async def zcount(
             ),
         )
 
+    async def zincrby(self, key: str, increment: float, member: str) -> float:
+        """
+        Increments the score of `member` in the sorted set stored at `key` by `increment`.
+        If `member` does not exist in the sorted set, it is added with `increment` as its score.
+        If `key` does not exist, a new sorted set with the specified member as its sole member is created.
```

Comment:
Darn.  Missed this on the Java side:  
```suggestion
        If `key` does not exist, a new sorted set is created with the specified member as its sole member.
```

Reply:
thanks!